### ISSUE TITLE
First Tier Electrical Transmission Lines Need A Make Over!

### ISSUE THREAD

**Issue by pooing1234** [Electrical-Age/ElectricalAge#754]
In all honesty, they look rushed, and really don't look pleasant at all. They are too low and have to be propped up so the cables can't be reached, or clip with the ground (as shown in the pictures).
The cables are too thick and make the transmission poles look a little weird.
It would be really nice to have some type of customization system.

Apart from that, since they've existed they have solved many problems I was having.
http://imgur.com/a/2QMc0

**Comment by AMIDIBOSS** [comment]
Dual height poles? Nice thinking!

**Comment by pooing1234** [comment]
@AMIDIBOSS Thanks. Needed to stack them so I could use the full capacity from the large turbine power station I'm working on.

**Comment by AMIDIBOSS** [comment]
Good thinking

**Comment by AMIDIBOSS** [comment]
But you wouldn't need it when t2 poles come to play... Not soon though...

**Comment by pooing1234** [comment]
@AMIDIBOSS Yeah, It's a pain to use them, especially on a server with my sister. She's built multiple farming facilities, and each building pulls around 20KW each. The city we're working on can pull up to 60 KW, more when I have friends online.
